### ISSUE TITLE
[A11Y] Registration form select list default text is announced as an alert

### ISSUE THREAD

**Issue by foxbunny** [indico/indico#6061]
**Describe the bug**

As soon as the page is loaded, all default texts on select lists are announced as alerts when using a screen reader.

**To Reproduce**
1. Go to a registration form that includes select lists
2. Observe that the select lists are all announced as alerts before the page is read

**Expected behavior**
The default text is not announced until the field is focused.

**Additional context**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html
